@navikt/core/react/src/modal/Modal.tsx

```diff
@@ -82,6 +82,7 @@ export const Modal = forwardRef<HTMLDialogElement, ModalProps>(
       open,
       onBeforeClose,
       onCancel,
+      closeOnClickOutside,
```

Comment:
`ClickOutside` beskriver kanskje ikke så bra hva `outside` er. Kunne f.eks `closeOnOverlayClick` fungert bedre?

Reply:
Ser den, men foreslår da "Backdrop" i stedet for "Overlay" siden det er det "offisielle" navnet.

Reply:
Enig, `Backdrop` fungerer bra 👍

Reply:
Fikset